eng/pipelines/publish.yml

```diff
@@ -46,7 +46,7 @@ jobs:
                 /t:PublishPackagesToBlobFeed
                 /p:ManifestBuildId=$(Build.BuildNumber)
                 /p:ManifestBuildData=Location=$(_dotnetFeedUrl)
-                /p:ManifestBranch=$(Build.SourceBranchName)
+                /p:ManifestBranch=$(Build.SourceBranch)
```

Comment:
dotnet/versions is also affected, updating `build-info/dotnet/corefx/3.0` at the moment. `release/3.0` is normal, there. I'm not sure it needs to be changed, though--it seems unlikely corefx will start producing some other branch that also ends in `/3.0`.

Reply:
The Tizen folks may care about the inconsistent naming, or at least need to know it's different so they can trigger their builds.

Reply:
I didn't want to mess with it too much.  AFAIK, there's no way to the string.replace to strip away the refs/heads in yml. I think leaving it as 3.0 is probably fine for now.